diagnostics/flow_dep_diags/doc/flow_dep_diags.rst

```diff
@@ -0,0 +1,136 @@
+.. This is a comment in RestructuredText format (two periods and a space).
```

Comment:
The commented sections can be removed from this file.

Reply:
@jkrasting could you point me to who I should contact regarding submission of the data for this POD?